types/recharts/index.d.ts

```diff
@@ -417,6 +417,9 @@ export interface LegendPayload {
     id: any;
     type: LegendType;
     color?: string;
+    payload?: {
+        strokeDasharray: string;
```

Comment:
forgive me if I missed it reading quickly, but it'd be great to have documentation or source code and not the issue you linked.

one reason why is that my next question after that is going to be "what else should be in this payload field" and having the source would help answer that.